### PR TITLE
Axis names fix

### LINKED ISSUE #479: make the option to disable overlaying more discoverable
This is a suggestion after trying to explain to a Falcon user how to create a subplot (see https://github.com/plotly/falcon-sql-client/issues/449).

The step I found hard to discover was how to disable overlaying:

screenshot from 2018-06-07 10-49-58

How do you think about having the drop-down menu show `disabled` as an option?

screenshot from 2018-06-07 11-00-29

### PR DESCRIPTION
**Pull request by dmt0**
addresses #575 
closes #479 

- refactored axes component naming
- correct axes name during creation
- axes names in axes panel
- add None option for Overlay

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
@VeraZab can you review this as well please, given that you spent so much time working on this initially? :)

File: src/components/fields/AxesSelector.js [hunk 1]
```diff
@@ -16,36 +16,45 @@ class AxesSelector extends Component {
...
-      return (
-        <Field {...this.props} label={_('Axis to Style')}>
-          <Dropdown
-            options={axesOptions.map(option => {
-              if (option.value !== 'allaxes') {
-                return {
+    const multipleSublots =
+      fullLayout &&
+      fullLayout._subplots &&
+      Object.values(fullLayout._subplots).some(s => s.length > 1);
```
Comment (VeraZab) [inline src/components/fields/AxesSelector.js]: hmm, fullLayout._subplots is an object right
that contains subplotTypes as keys, and arrays as values, and we want to check that those arrays are of length greater than 1.

so isn't it more like:
`Object.values(fullLayout._subplots).some(s => fullLayout[s].length > 1);`

**Comment by VeraZab** [comment]
other than the above, looks good to me!

File: src/components/fields/AxesSelector.js (hunk 1 above)
Comment (dmt0) [reply src/components/fields/AxesSelector.js]: Vera, it's Object.values(), not Object.keys()

**Comment by VeraZab** [comment]
💃

**Comment by nicolaskruchten** [comment]
can you force-push here to re-trigger circle and percy plz?

**Comment by dmt0** [comment]
CircleCI is still down as of 6 minutes ago

**Comment by dmt0** [comment]
I'm thinking Overlay should have an explicit option for "none" - it took me a while to figure out that I have to clear it to spread out the plots.

**Comment by nicolaskruchten** [comment]
Agreed, please add 'none'. This will close #479

**Comment by dmt0** [comment]
err, am I still passing review after all these commits? :)

**Comment by nicolaskruchten** [comment]
@VeraZab please re-review if needed! I think the behaviour is fine for me. The only known-weird thing left outstanding with the secondary y-axis thing is https://github.com/plotly/plotly.js/issues/2828

**Comment by VeraZab** [comment]
ok rechecked looks good 💃

**Pull request merged**